### PR TITLE
Added new personAttributeTypes for registration form

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: constants.yml [hunk 1]
```diff
@@ -6,3 +6,11 @@ patientIdentifierType:
     uuid: "ffbd60fb-599a-4a57-b2f1-4b55847cd938"
   Passport:
     uuid: "3fa0990e-900d-4a80-a4c1-dcf01a0d4f9c"
+  Email:
+    uuid: "ade498b7-99a5-4cca-891f-679702a802ff"
```
Comment (brandones) [inline constants.yml:10]: Email isn't a PIT, is it?

Comment (MiguelAHPpih) [reply constants.yml:10]: COR uses mainly document number as an a identifier. Making mail more of an attribute than an identifier.

Comment (brandones) [reply constants.yml:10]: Right so let's delete these two lines from the code?

Comment (MiguelAHPpih) [reply constants.yml]: Sorry for the delay. @brandones , done!

**Review by brandones — APPROVED** [review]

**Pull request merged**